data-serving/data-service/src/model/case.ts

```diff
@@ -39,12 +39,16 @@ const caseSchema = new mongoose.Schema(
             type: revisionMetadataSchema,
             required: 'Must include revision metadata',
         },
-        notes: String,
+        notes: {
+            type: String,
+            text: true,
+        },
         pathogens: [pathogenSchema],
         preexistingConditions: dictionarySchema,
         sources: {
             type: [sourceSchema],
             required: true,
+            text: true,
```

Comment:
by defining this at this level, does it apply to all fields of the sources sub-schema?

Reply:
by defining this at this level, I've shown the world that I am stupid. This was a mistake.